charts/private-link/values.yaml

```diff
@@ -0,0 +1,171 @@
+################################################################################
+# Global options
+################################################################################
+global:
+  image:
+    # global image pull policy to use for all container images in the chart
+    # can be overridden by individual image pullPolicy
+    pullPolicy:
+    # global registry to use for all container images in the chart
+    # can be overridden by individual image registry
+    registry: registry.synadia.io
+
+  # global labels will be applied to all resources deployed by the chart
+  labels: {}
+
+################################################################################
+# Private Link Deployment and associated resources
+################################################################################
+
+############################################################
+# image pull secret
+############################################################
+# when enabled, the registry for images in the private link deployment
+# will default to global.image.registry
+imagePullSecret:
+  enabled: false
+  registry:
+  username:
+  password:
```

Comment:
The image will be public by default, so I don't think we need a quick way to manage pull secrets